package.json

```diff
@@ -4,13 +4,13 @@
   "main": "index.html",
   "repository": "git@github.com:newhavenio/newhavenio.github.io.git",
   "contributors": [
-     "Brady Sutton <bsutt123@gmail.com>"
-     "Dan Bernier <danbernier@gmail.com>"
+     "Brady Sutton <bsutt123@gmail.com>",
+     "Dan Bernier <danbernier@gmail.com>",
      "Joel Nimety <jnimety@gmail.com>",
-+    "Josh Santos <josh@omnidapps.com>",
-+    "maxx1128 <maxx1128@gmail.com>",
-     "Zach Morek" <zmorek@gmail.com>"
-+  ],
+     "Josh Santos <josh@omnidapps.com>",
+    "maxx1128 <maxx1128@gmail.com>",
+     "Zach Morek <zmorek@gmail.com>"
+  ],
```

Comment:
I fixed a lot of this on master. Maybe do a rebase?

Reply:
+1 on this, also, for anything that isn't fixed, we should make sure the whitespace is consistent